src/Account.js

```diff
@@ -69,13 +69,23 @@ class Account {
   }
 
   // Used for notes, medication reports, etc
-  create(path, attrs, uuid) {
+  create(path, attrs) {
     return this.network.run`
-      ${this.model}.find_by(uuid: '${uuid}').
+      ${this.model}.find_by(uuid: '${this.information.uuid}').
         ${path}.
         create!(JSON.parse('${JSON.stringify(attrs)}'))
     `
   }
+
+  // Used for notes, medication reports, etc
```

Comment:
Are we going to be forgetting other reports in the future?

Reply:
I like the idea of being able to delete a report that was accidentally submitted. Maybe a future goal.